Buildings/Experimental/OpenBuildingControl/ASHRAE/G36/AtomicSequences/EconModulationMultiZone.mo

```diff
@@ -0,0 +1,194 @@
+within Buildings.Experimental.OpenBuildingControl.ASHRAE.G36.AtomicSequences;
+block EconModulationMultiZone
+  "Outdoor and return air damper position modulation sequence for multiple zone VAV AHU"
+
+  parameter Real outDamConSigMax(min=0, max=1, unit="1") = retDamConSigMin
+  "Maximum control loop signal for the outdoor air damper";
+  parameter Real retDamConSigMin(min=0, max=1, unit="1") = 0.5
+  "Minimum control loop signal for the return air damper";
+  parameter Real yConSigMin=0 "Lower limit of controller output";
+  parameter Real yConSigMax=1 "Upper limit of controller output";
+  parameter Real controllerGain=1 "Gain of controller";
```

Comment:
Please rename `controllerGain` to `k` (or `kP` if it is a proportional gain and you have other gains as well.)